src/main/java/com/coveros/selenified/element/check/AssertEquals.java

```diff
@@ -72,6 +72,18 @@ public void matches(int expectedMatches) {
         assertEquals("Element Match Mismatch", expectedMatches, checkMatches(expectedMatches, 0, 0));
     }
 
+    /**
+     * Asserts that the element's tag name equals the provided expected tag name. If
+     * the element isn't present, this will constitute a failure, same as a
+     * mismatch. This information will be logged and recorded, with a screenshot
+     * for traceability and added debugging support.
+     */
+    public void tagName(String expectedTagName) {
+        String tagName = checkTagName(expectedTagName, 0, 0);
+        assertNotNull(NO_ELEMENT_FOUND, tagName);
+        assertEquals("Text Mismatch", expectedTagName, tagName);
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Define a constant instead of duplicating this literal "Text Mismatch" 3 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://localhost:9000/sonar/coding_rules#rule_key=squid%3AS1192)